conda_build/post.py

```diff
@@ -1732,6 +1732,54 @@ def fix_permissions(files, prefix):
                 log.warn(str(e))
 
 
+def check_menuinst_json(files, prefix):
+    json_files = fnmatch_filter(files, "[Mm]enu[/\\]*.[Jj][Ss][Oo][Nn]")
+    if not json_files:
+        return
+
+    print("Validating Menu/*.json files")
```

Comment:
Would it be a better idea to make this a log.info call? I'm always a little skeptical of print statements 🤔.

Reply:
Just following trend with the rest of the module :( Search for `print` in `post.py` and be amazed 😬